Description: Remove bundler dependency
 Remove bundler dependency from the test_helper file.
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-01-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,4 +1,3 @@
-require 'bundler/setup'
 require 'minitest/autorun'
 require 'active_model_serializers'
 require 'fixtures/poro'
